server/src/GameState.ts

```diff
@@ -0,0 +1,21 @@
+import { EntityMap, Client } from "colyseus";
+import Player from "./Player";
+import Rotation from "./Rotation";
+import Position from "./Position";
+
+export class GameState {
+    players: EntityMap<Player> = {};
+
+    addPlayer(client: Client) {
+        this.players[client.sessionId] = new Player();
+    }
+
+    removePlayer(client: Client) {
+        delete this.players[client.sessionId];
+    }
+
+    movePlayer(client: Client, data: {position: Position, rotation: Rotation}) {
```

Comment:
for now I would not validate player position and rotation, as the player shouldn't just say what his new position is, but instead say what were his inputs (as in - moving left, right, forward and backward), and the game should calculate his new position based on the inputs and the player stats (when there will be a stat such as movement speed)

Reply:
Done.